data/Junnygram.json

```
@@ -0,0 +1,45 @@
+{
+  "name": " Oyewunmi Olaleye",
+  "type": "personal",
+  "bio": "**Frontend developer!** | Physicist ",
+  "tags": [
+    "Open Source",
+    "Javascript",
+    "Python",
+    "web Development",
```

You can just capitalize 'w' in web Development. Otherwise, everything looks good.
```suggestion
    "Web Development",
```

---

Good spot 👍 , this is not a blocker though, as further improvements could come in another pull request